components/account/AccountPostsFollowers.vue

```diff
@@ -56,6 +58,7 @@ const userSettings = useUserSettings()
       :to="getAccountFollowersRoute(account)"
       replace text-secondary
       exact-active-class="text-primary"
+      data-glean="profile.details.followers"
```

Comment:
I'm not able to get these `profile.details.*` events to fire in the console. I only see the `link_click` following by `page_view` event. Not seeing them log to the Glean ping debugger either.

Reply:
![image](https://github.com/MozillaSocial/elk/assets/14023085/b590ffdf-00d6-4091-8da1-031336008d38)
`<a>` current require you to click on them directly instead of any nested children

Reply:
Whoops this screenshot is for a different UI. But similar reason, the <a> needs to be clicked directly instead of their children - 
<img width="433" alt="image" src="https://github.com/MozillaSocial/elk/assets/14023085/fe5a19ab-65ba-48df-be43-795de200b885">

Reply:
Sounds good to me. We can solve this with something applicable to all anchor tags